src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -72,6 +74,19 @@ object ECDSASignature {
     else s
   }
 
+  def sign(message: Array[Byte], keyPair: AsymmetricCipherKeyPair): ECDSASignature = {
+    val signer = new ECDSASigner(new HMacDSAKCalculator(new SHA256Digest))
+    signer.init(true, keyPair.getPrivate)
+    val components = signer.generateSignature(message)
+    val r = components(0)
+    val s = ECDSASignature.canonicalise(components(1))
+    val v = ECDSASignature
+      .calculateV(r, s, keyPair, message)
+      .getOrElse(throw new RuntimeException("Failed to calculate signature rec id"))
```

Comment:
Should this exception be handled or it will be handled by the user of it?
As so far this is used only for testing this is not something urgent to solve if it needs to be.